### ISSUE TITLE
rest-core-0.39.0.1 failed to build to lts-11

### ISSUE THREAD

**Issue by juhp** [silkapp/rest#158]
```
rest-core-0.39.0.1/src/Rest/Driver/RestM.hs:53:10: error:
        Not in scope: type constructor or class ‘Semigroup’
       |
    53 | instance Semigroup RestOutput where
       |          ^^^^^^^^^
```

**Comment by hesselink** [comment]
@bergmark Seems you changed this recently?

**Comment by bergmark** [comment]
ah we aren’t using base gompat there, i can fix it today

**Comment by bergmark** [comment]
Was fixed in 0.39.0.2
